Sources/Purchasing/Configuration.swift

```diff
@@ -392,7 +392,14 @@ extension Configuration {
         #if TEST_STORE
         if apiKey.hasPrefix(testStoreKeyPrefix) {
             // Test Store key format: "test_CtDdmbdWBySmqJeeQUTyrNxETUVkajsJ"
+
+            #if DEBUG
             return .testStore
+            #else
+            // In release builds, we intentionally crash to prevent submitting an app with a Test Store API key
+            fatalError("[RevenueCat]: Test Store API key used in RELEASE build." +
+                       "Please configure a valid API key before releasing.")
```

Comment:
Should we guide the dev a bit more? Maybe: 
```suggestion
                       "Please configure the App Store app on the RevenueCat dashboard and use its corresponding Apple API key before releasing.")
```

Reply:
I like it! yes, it's much clearer this way 👌 Thanks!!
Done in https://github.com/RevenueCat/purchases-ios/pull/5453/commits/9257137401bbf15d518fbed7eb9c01e82cdfa79f